### PR TITLE
add tenant admin clarification

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #646

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/manage/centralized-deployment-of-add-ins.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Import-Module O365CompatibilityChecker
 ```powershell
 Invoke-CompatibilityCheck
 ```
-   which prompts you for  *_TenantDomain_* (for example, *TailspinToysIncorporated.onmicrosoft.</span>com*) and  *_TenantAdmin_* credentials, and then requests consent.
+   which prompts you for  *_TenantDomain_* (for example, *TailspinToysIncorporated.onmicrosoft.</span>com*) and  *_TenantAdmin_* credentials (Global admin credentials must be entered), and then requests consent.
```
Comment (nenonix) [inline Office365-Admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
   which prompts you for  *_TenantDomain_* (for example, *TailspinToysIncorporated.onmicrosoft.</span>com*) and  *_TenantAdmin_* credentials (use your global admin sign in), and then requests consent.
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix ,
your suggestion was applied,
thank you!

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #646.

Thank you.

**Comment by CrystalThomasMS** [comment]
@Sirkkuw OK to merge?

**Comment by Sirkkuw** [comment]
I am ok with this change @mypil

**Pull request merged**